website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -7,6 +7,10 @@ page_id: api-changelog
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-03-13
+
+* Update [Project Team Access API](/terraform/cloud-docs/api-docs/project-team-access) to include additional Project roles
```

Comment:
Very nit-picky, feel free to ignore, looks like you can use full stop just to be consistent with most of the change logs.